releasenotes/notes/fix_ci_fails_due_to_python311-367ea7ca0abbfc2a.yaml

```diff
@@ -0,0 +1,5 @@
+---
+fixes:
+  - |
+    Fix the minor version of Python 3.11 in CI to work around bugs of terra for 3.11.1.
+    Also, change installation of Qiskit-Aeer in Neko to correctly install `qiskit_aer` package.
```

Comment:
We don't need a release note for this change, CI changes aren't user facing (which is the intended audience of release notes).